webapp/src/components/common/DataGridViewer.tsx

```diff
@@ -72,8 +82,10 @@ function DataGridViewer({ data, columns }: DataGridViewerProps) {
       rows={data.length}
       columns={columns}
       getCellContent={getCellContent}
-      rowMarkers="none"
+      rowMarkers={rowMarkers}
       getCellsForSelection
+      freezeColumns={fixedFirstColumn ? 1 : 0}
```

Comment:
Why not using `freezeColumns` in props instead of the limited `fixedFirstColumn` prop?